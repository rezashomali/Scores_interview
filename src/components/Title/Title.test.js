import React from "react";
import renderer from "react-test-renderer";
import Title from "./Title";

describe("Title", () => {
  test("renders properly", () => {
    const tree = renderer.create(<Title>title</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
