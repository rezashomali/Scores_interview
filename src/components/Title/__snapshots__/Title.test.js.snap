// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders properly 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
>
  title
</h2>
`;
